include/libswiftnav/signal.h

```diff
@@ -19,15 +20,35 @@
 #define NUM_SATS_SBAS 19
 #define NUM_SATS (NUM_SATS_GPS + NUM_SATS_SBAS)
 
+#define CODES_ON_GPS_SAT 2
+#define CODES_ON_SBAS_SAT 1
+
+#define NUM_CODES_GPS (NUM_SATS_GPS * CODES_ON_GPS_SAT)
```

Comment:
Sounds reasonable, although at the moment L2CL is not really supported, it just ended up into the code enum for some reason. Maybe it should be still omitted at this point.